Doc/library/asyncio-stream.rst

```diff
@@ -18,17 +18,16 @@ streams::
     import asyncio
 
     async def tcp_echo_client(message):
-        reader, writer = await asyncio.open_connection(
-            '127.0.0.1', 8888)
+        stream = await asyncio.connect('127.0.0.1', 8888)
 
         print(f'Send: {message!r}')
-        await writer.write(message.encode())
+        stream.write(message.encode())
```

Comment:
```suggestion
        await stream.write(message.encode())
```
Missing `await` is supported but discouraged.

Reply:
Thanks for the suggestion. Changed in this example and in others to make sure write calls are awaited.